### ISSUE TITLE
Clear Detector sample info after finishing of acquisition 

### ISSUE THREAD

**Issue by bdrum** [regata-jinr/MeasurementsUI#37]
After measurements old information about sample saved on detector. It can make a mess.

**Comment by bdrum** [comment]
Here is the problem with clearing the irradiation DateTime fields.
Such code
~~~csharp
d.Param[ParamCodes.CAM_X_SDEPOSIT] = string.Empty;
d.Param[ParamCodes.CAM_X_STIME] = string.Empty;
~~~ 

 throws exception:

~~~
 Error: ece99d7c. Windows System Error: Cannot convert BSTR to CSTR.
~~~

A possible solution is to create empty spectra (save file after new detector added) and run such code:

~~~csharp
            DeviceAccessClass d = null;
            DataAccessClass f = null;
            try
            {
                d = new DeviceAccessClass();
                d.Connect("D2");
                f = new DataAccessClass();
                f.Open(@"C:\GENIE2K\empty_spectra.cnf");
                f.CopyBlock(d, CanberraDataAccessLib.ClassCodes.CAM_CLS_SAMP);
                d.Save("", true);
            }
            finally
            {
                d?.Disconnect();
                f?.Close();
            }
~~~

**Comment by bdrum** [comment]
Closed in 8b0749d
